src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceDropdown.html

```diff
@@ -0,0 +1,96 @@
+<div class="dropdown resource-header-actions" data-ng-if="!data.modals.modal && data.isShowDropdownMenu">
```

Comment:
`data.isShowDropdownMenu` will come into play in 9d8a475 (make dropdown button disappear when alerts appear). sorry for the confusion.